ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -144,5 +132,31 @@ public void WhenTwoUniversesInstantiated_BeEqual()
 
             Assert.True(result);
         }
+
+        [Fact]
+        public void WhenTwoUniversesInstantiated_NotBeEqual()
+        {
+            var gridWidth1 = 4;
+            var gridLength1 = 4;
+            var gridWidth2 = 5;
+            var gridLength2 = 4;
+            var universe1 = new Universe(gridWidth1, gridLength1);
+            var universe2 = new Universe(gridWidth2, gridLength2);
+
+            var result = UniverseHelper.UniversesAreEqual(universe1, universe2);
+
+            Assert.False(result);
+        }
+
+        [Fact]
+        public void WhenCellIsNull_ThrowException() // TODO: rename - if cell in GetCellNeighbourLocations is null & throws exception
+        {
+            Cell cell = null;
+            var universe = new Universe(3, 3);
+
+            var result = Assert.Throws<NullReferenceException>(() => universe.GetCellNeighbourLocations(cell));
+            
+            Assert.Equal("Object reference not set to an instance of an object.", result.Message);

```

Comment:
My opinion: I don't think you need to test the error message specifically as they can be customised/changed, so you don't want to have to update your test if you decide to modify your error message. It's enough to test that the null reference error will be thrown if the cell is null. Also when asserting that an error is thrown, you don't have to assign this to a variable.